cmd/clusterctl/cmd/config_repositories.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/cluster-api/cmd/clusterctl/pkg/client"
+)
+
+var configRepositoryCmd = &cobra.Command{
+	Use:   "repositories",
```

Comment:
I don't know how confusing it would be but we could name this sub-command `providers`. That is, distinguish between singular and plural.
The user experience would be
```bash
# this would essentially this command.
clusterctl config providers
# and this would be the one above.
clusterctl config provider aws -o yaml
```
That way we don't have to add another vocabulary word that needs clarification.
WDYT?

Reply:
The meaning of repository is well documented in clusterctl docs, so while implementing the PR I decided that using this term as a command name can represent better what the command does and it removes any possible confusion between the two commands.
But I have no strong opinions and I will adapt to what the consensus decides

Reply:
Likewise, I'm flexible with whatever the consensus is. I did notice the "provider repository" terminology is already in the docs. So it makes sense.